expressions/methods.md

```diff
@@ -196,6 +196,19 @@ class Foo
 
 __Can I call using positional arguments but miss out the first one?__ No. If you use positional arguments they must be the first ones in the call.
 
+## Chaining
+
+Method chaining allows you to chain calls on an object without requiring the method to return its receiver. The syntax to call a method and chain the receiver is `object.>method()`.
+
+```pony
+primitive Printer
+  fun print_two_strings(out: StdStream, s1: String, s2: String) =>
+    out.>print(s1).print(s2)
```

Comment:
Hmm I would have expected to do

`out.print(s1).>print(s2)`

Clarification and more than 1 example would definitely be good.